File: library/camera/src/main/java/com/suishi/camera/camera/init/CameraInfo.java
package com.suishi.camera.camera.init;

import android.util.Size;

public class CameraInfo {
    private String name;
    private String cameraId;
    private Size size;
    private int fps;

    public CameraInfo(String name, String cameraId, Size size, int fps) {
        this.name = name;
        this.cameraId = cameraId;
        this.size = size;
        this.fps = fps;
    }

    public String getName() {
        return name;
    }

    public String getCameraId() {
        return cameraId;
    }

    public Size getSize() {
        return size;
    }

    public int getFps() {
        return fps;
    }
}

File: app/src/main/java/com/suishi/live/app/modle/CameraInfo.kt
package com.suishi.live.app.modle

import android.util.Size


class CameraInfo {

    constructor(name: String?, cameraId: String?, size: Size?, fps: Int) {
        this.name = name
        this.cameraId = cameraId
        this.size = size
        this.fps = fps
    }

    var name:String?=null
    var cameraId:String?=null
    var size: Size?=null
    var fps:Int=0

}